class.jetpack-heartbeat.php

```diff
@@ -116,7 +116,6 @@ public static function generate_stats_array( $prefix = '' ) {
 		$return["{$prefix}is-multisite"]   = is_multisite() ? 'multisite' : 'singlesite';
 		$return["{$prefix}identitycrisis"] = Jetpack::check_identity_crisis() ? 'yes' : 'no';
 		$return["{$prefix}plugins"]        = implode( ',', Jetpack::get_active_plugins() );
-		$return["{$prefix}manage-enabled"] = Jetpack::is_module_active( 'manage' );
```

Comment:
Makes sense.  Don't think we really use this stat but won't hurt to keep it accurate :)